800.renames-and-merges/d.yaml

```diff
@@ -92,6 +92,7 @@
 - { setname: diskonaut,                name: "rust:diskonaut" }
 - { setname: distcc,                   name: [distcc-gnome,distcc-gtk,distcc-monitor,distcc-nozeroconf,distcc-pump,distcc-server], addflavor: true }
 - { setname: distorm,                  name: [distorm3, distorm64, "python:distorm", "python:distorm3" ] }
+- { setname: distribution-cncf,        name: docker-distribution }
```

Comment:
Please merge `docker-distribution` into `distribution` instead

It's more convenient and less error prone to make all packages follow the same ruleset path - that is everything is merged into a single name, then split. It requires less rules, and if we want to change the target name, it may be done in single place